src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -98,7 +98,7 @@ protected override void ApplyResponseGrant()
                     string redirectUri = notification.ProtocolMessage.CreateLogoutRequestUrl();
                     if (!Uri.IsWellFormedUriString(redirectUri, UriKind.Absolute))
                     {
-                        _logger.LogWarning("The logout redirect URI is malformed: {0}", (redirectUri ?? "null"));
+                        _logger.LogWarning(Resources.OIDCH_0051_RedirectUriLogoutIsNotWellFormed, (redirectUri ?? "null"));
```

Comment:
I agree with there may be a bug somewhere, which we may or may not have caught before shipping, which may or may not cause an issue or failure. I have found that developing with a defensive attitude helps with the user experience and helps avoid low grade errors. At the end of the day customer experience rules.